.github/workflows/perf-ci.yml

```diff
@@ -0,0 +1,35 @@
+name: Performance Benchmarking CI
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  perf:
+    name: Perfharness
+    runs-on: ubuntu-22.04
+    container:
+      image: angr/ci:3
+      options: --privileged
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        repository: angr/angr-dev
+    - name: Install
```

Comment:
We should use uv and make a benchmarking/codspeed dep group, instead of using angr-dev. See https://github.com/angr/angr/blob/master/.github/workflows/nightly-ci.yml for what this looks like

Reply:
Oh I see it was added to the dev group, an extra benchmarking group isn't necessary then